streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/config/types/SlotAssignStrategyType.java

```diff
@@ -0,0 +1,24 @@
+package org.ray.streaming.runtime.config.types;
+
+/**
+ * Slot assign strategy type.
```

Comment:
It makes no sense to turn the class name into a sentence. Please describe the purpose of this class.

Reply:
Added description information to specific schedule strategy.